chart/skywalking/files/conf.d/oap/README.md

```diff
@@ -0,0 +1,11 @@
+If you don't want to use the default `application.yml` packed into the Docker image,
```

Comment:
What is this doc for? Which doc linking to here?

Reply:
Git cannot add an empty directory, this file is just to make `oap` not empty and can be checked into the codebase

Reply:
I just remove it and add a `.gitignore` to do the same thing, and ignore the testing files locally